packages/wallets/src/features/cashier/flows/WalletWithdrawal/WalletWithdrawal.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
-import { WithdrawalFiatModule } from '../../modules';
+import React, { useState } from 'react';
+import { WithdrawalFiatModule, WithdrawalVerificationModule } from '../../modules';
 
 const WalletWithdrawal = () => {
-    //TODO: add withdrawal crypto module
-    return <WithdrawalFiatModule />;
+    const queryParams = new URLSearchParams(location.search);
+    const verificationQueryParam = queryParams.get('verification');
+
+    const [verificationCode, setVerificationCode] = useState('');
+
+    if (verificationQueryParam) {
+        setVerificationCode(verificationQueryParam);
+
+        const url = new URL(window.location.href);
+        url.searchParams.delete('verification'); // Remove the 'verification_code' query parameter
+        window.history.replaceState({}, document.title, url.toString());
```

Comment:
Should this be in a `useEffect` call?

Reply:
hey, updated @adrienne-deriv

Reply:
also as I have been updating it, I've realised that routing probably should be set up differently, so its likely that I'll raise another PR for that, to make it simpler